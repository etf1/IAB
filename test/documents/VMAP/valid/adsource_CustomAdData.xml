<!-- AdSources with different CustomAdData and templateType values-->
<vmap:VMAP version='1.0' xmlns:vmap='http://www.iab.net/vmap-1.0'>
    <vmap:AdBreak breakId='cue_point-347762684' breakType='linear' timeOffset="start" >
        <vmap:AdSource allowMultipleAds='false' followRedirects='true' id='1' >
            <vmap:CustomAdData templateType="vast1">
                <![CDATA[http://7b9df.v.fwmrm.net/ad/1]]>
            </vmap:CustomAdData>
        </vmap:AdSource>
    </vmap:AdBreak>
    <vmap:AdBreak breakId='cue_point-347762684' breakType='linear' timeOffset="start" >
        <vmap:AdSource allowMultipleAds='false' followRedirects='true' id='1' >
            <vmap:CustomAdData templateType="vast2">
                <tag><root>1</root></tag>
            </vmap:CustomAdData>
        </vmap:AdSource>
    </vmap:AdBreak>
    <vmap:AdBreak breakId='cue_point-347762684' breakType='linear' timeOffset="start" >
        <vmap:AdSource allowMultipleAds='false' followRedirects='true' id='1' >
            <vmap:CustomAdData templateType="proprietary">hello, world</vmap:CustomAdData>
        </vmap:AdSource>
    </vmap:AdBreak>
    <vmap:AdBreak breakId='cue_point-347762684' breakType='linear' timeOffset="start" >
        <vmap:AdSource allowMultipleAds='false' followRedirects='true' id='1' >
            <vmap:CustomAdData templateType="proprietary">
                <VAST version='3.0'></VAST>
            </vmap:CustomAdData>
        </vmap:AdSource>
    </vmap:AdBreak>
</vmap:VMAP>
