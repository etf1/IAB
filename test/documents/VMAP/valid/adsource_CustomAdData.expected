{
  "version": "1.0",
  "breaks": [
    {
      "timeOffset": "start",
      "breakTypes": [
        1
      ],
      "source": {
        "id": "1",
        "allowMultipleAds": false,
        "followRedirects": true,
        "dataType": 2,
        "adDataType": 1,
        "customAdData": "http://7b9df.v.fwmrm.net/ad/1"
      },
      "id": "cue_point-347762684"
    },
    {
      "timeOffset": "start",
      "breakTypes": [
        1
      ],
      "source": {
        "id": "1",
        "allowMultipleAds": false,
        "followRedirects": true,
        "dataType": 2,
        "adDataType": 2,
        "customAdData": {
          "templateType": "vast2",
          "tag": [
            {
              "root": [
                {
                  "$t": "1"
                }
              ]
            }
          ]
        }
      },
      "id": "cue_point-347762684"
    },
    {
      "timeOffset": "start",
      "breakTypes": [
        1
      ],
      "source": {
        "id": "1",
        "allowMultipleAds": false,
        "followRedirects": true,
        "dataType": 2,
        "adDataType": 3,
        "customAdData": "hello, world"
      },
      "id": "cue_point-347762684"
    },
    {
      "timeOffset": "start",
      "breakTypes": [
        1
      ],
      "source": {
        "id": "1",
        "allowMultipleAds": false,
        "followRedirects": true,
        "dataType": 2,
        "adDataType": 3,
        "customAdData": {
          "templateType": "proprietary",
          "VAST": [
            {
              "version": "3.0"
            }
          ]
        }
      },
      "id": "cue_point-347762684"
    }
  ]
}
